### DISCUSSION TITLE
Changing the App name request

### DISCUSSION THREAD

**Discussion by bay127prem** [smol-ai/GodMode#161] [General]
Hello,

As you may know, “God” holds great significance and reverence for many people around the world. Using this word in the name of an app may be considered inappropriate or even offensive.

I understand that it may not have been your intention to cause any offense, and I appreciate the hard work that has gone into creating this app. However, I kindly request that you consider changing the name of the app to something more suitable.

**Comment by abdurahman-harouat** [comment]
i do agree!

**Comment by dportela** [comment]
Yes, while I understand the gaming reference, a name that doesn't offend 50% of the world will speed up adoption.
